config/initializers/sentry.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Sentry.init do |config|
+  config.dsn = 'https://e8bd9a43b666488db6bee4975626669d@o949860.ingest.sentry.io/5898547'
```

Comment:
You shouldn't push data like this to your GitHub repository. You can use for example dotenv to mange environmental variables :).

Reply:
thanks, I added dotenv gem :)